tests/test_save.py

```diff
@@ -0,0 +1,57 @@
+import os
+import shutil
+
+import numpy as np
+
+import btk
+from btk.survey import Rubin
+
+
+def test_save():
+    try:
+        shutil.rmtree("./output")
```

Comment:
I suggest not to use "output" as the test directory since it would be a default choice for a given user to name an actual output repo. Maybe something like `"output_test_dir".

Also I would like to point out a useful library for these kind of things : [tempfile](https://docs.python.org/3/library/tempfile.html)